### PR TITLE
Countries App

### PR DESCRIPTION
**Pull request by RayhanTabase**
## Countries App: React Capstone
In this feature:
- Built a React app using the Countries API to fetch data about countries all around the world and displayed the data.
- Used a SPA design
- Used routing
- Used duck pattern
- Used Redux
- Used tailwind CSS for styling
- Presentation; [Video site usage](https://www.loom.com/share/ccd0d737d0c2448697508d3aab9160c8?sharedAppSource=personal_library) and [Video interesting code](https://www.loom.com/share/ccd0d737d0c2448697508d3aab9160c8?sharedAppSource=personal_library)

### CODE REVIEW

File: src/components/Continent.jsx [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+    case 'Europe':
+      marked = [];
+      marked.push({ country: 'ru', value: 1389618778 });
+      marked.push({ country: 'ru', value: 210797836 });
+      break;
+
+    case 'Africa':
+      marked = [];
+      // marked.push(data[0]);
```
Comment (Meltrust) [inline src/components/Continent.jsx]: - [ ] Please kindly please remove this old commented piece of code.  That way, your code will be a lot cleaner 👍

File: src/index.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { Provider } from 'react-redux';
+
+// import reportWebVitals from './reportWebVitals';
```
Comment (Meltrust) [inline src/index.js]: - [ ] Please kindly please remove this old commented piece of code.  That way, your code will be a lot cleaner 👍

**Review by Meltrust — CHANGES_REQUESTED** [review]
Hi @RayhanTabase, 

## Your project works and looks perfectly and like a dream! Just mind my small comments below 👇  in order to comply with the requirements and you should be approved really fast!! 💪  

## Nice pagination, btw 🥇 


Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Ghiftee — APPROVED** [review]
## STATUS: APPROVED ♻️♻️
Hi @RayhanTabase, 

Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀  
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

**Pull request merged**